examples/cloud_storage_stream_inference/README.md

```diff
@@ -0,0 +1,102 @@
+# Using fsspec to stream data from cloud storage providers for bacth inference
+
+This example shows how to use fsspec to stream large amount of data from cloud storage like s3, google cloud storage, azure cloud etc. and use it to create requests to torchserve for large scale batch inference with a large batch size.
+
+The example shows retrieval of data using s3 API and fsspec, but the same idea can be used to retrieve data from other cloud storage peovider like Google Cloud, Azure etc.
+**Requires python >= 3.8**
+
+## Example overview
+
+  - Main objective of this example is to show the process of reading data from a cloud storage provider and send batch inference request to a model deployed using torchserve. We're not concerned with the accuracy of predictions.
+  - In this example we're going to run pre-trained distilbert model with torchserve and do text classification by reading input data using S3 API and fsspec and making REST API call to torchserve for batch inference.
+  - We're going to use Amazon reviews dataset which contains customer reviews along with title and label.
+  - We'll be taking customer review text and predict if the review is positive or negative usign pre-trained distilbert model.
+  - We'll be using minio https://min.io/ as AWS S3 proxy to store Amazon reviews dataset tar file.
+
+## Steps
+
+1) Install minio -  https://hub.docker.com/r/minio/minio/
```

Comment:
Could you please add a sentence on what this is for

Reply:
Added MinIO info